### PR TITLE
Add release workflow

### PR DESCRIPTION
**Pull request by Jimver**
Fixes #[issue_no]

## Summary of Changes

This adds a github actions release workflow that automatically uploads the zip file when a release is created, so no more manual uploading necessary!

At my fork you can see how it looks: https://github.com/Jimver/a32nx/releases/tag/v0.4.1.2

The github actions page at my fork: https://github.com/Jimver/a32nx/actions/runs/351447219

Note that it doesn't trigger for pre-releases, so it won't overwrite your pre-release that is tied to master, I also tested this at my fork: https://github.com/Jimver/a32nx/releases/tag/V0.3.1.2

## Additional context

Discord username (if different from GitHub): jimver#8723

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build.py will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, slick on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Comment by Benjozork** [comment]
code LGTM

**Comment by Benjozork** [comment]
@AdenFlorian can you confirm this is good ?

**Comment by Benjozork** [comment]
We gotta update this to work with `npm run build` of #1502

**Comment by Jimver** [comment]
> 
> 
> We gotta update this to work with `npm run build` of #1502

@Benjozork I can do that! Should I pull that branch into mine and then modify it?

**Comment by Benjozork** [comment]
@Jimver sorry I didn't get back to you earlier, you can now rebase this onto master as #1502 is merged.

**Comment by Jimver** [comment]
@Benjozork Release workflow now uses npm as well, just like the master workflow!

**Comment by Jimver** [comment]
@Benjozork Just updated this PR with master. I did it through the GitHub UI though which is a merge instead of rebase I think, is this a problem?

**Comment by lukecologne** [comment]
Should be fine

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    env:
+      RELEASE_ZIP_NAME: flybywiresim-a32nx.zip
+      MOD_FOLDER_NAME: A32NX
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+      - name: Setup Python 3
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
```
Comment (nistei) [inline .github/workflows/release.yml:20]: This step isn't needed anymore and can safely be removed.

Comment (Jimver) [reply .github/workflows/release.yml]: Yes that's right, just removed it!

**Review by Benjozork — APPROVED** [review]

**Pull request merged**